ifrs17-template/Test/Data/IfrsVariableBenchmarks/BM_CH_2020_12_MTUP10pct_BE.csv

```diff
@@ -28,40 +28,58 @@ AccidentYear,AmountType,AocType,DataNode,EconomicBasis,EstimateType,Novelty,Valu
 ,CDR,EOP,DTR1.1,C,BE,C,0.01057
 ,CDR,BOP,DTR1.1,C,BE,N,0.04415
 ,CDR,CF,DTR1.1,C,BE,N,-0.03368
+,CDRI,EOP,DTR1.1,C,BE,C,0.01057
+,CDRI,BOP,DTR1.1,C,BE,N,0.04415
+,CDRI,CF,DTR1.1,C,BE,N,-0.03368
 ,NIC,EOP,DTR1.1,C,BE,C,-164.82155
 ,NIC,BOP,DTR1.1,C,BE,N,-329.31412
 ,NIC,CF,DTR1.1,C,BE,N,165
 ,NIC,IA,DTR1.1,C,BE,N,-0.50743
 ,CDR,EOP,DTR1.1,L,BE,C,0.01057
 ,CDR,BOP,DTR1.1,L,BE,N,0.04415
 ,CDR,CF,DTR1.1,L,BE,N,-0.03368
+,CDRI,EOP,DTR1.1,L,BE,C,0.01057
+,CDRI,BOP,DTR1.1,L,BE,N,0.04415
+,CDRI,CF,DTR1.1,L,BE,N,-0.03368
 ,NIC,EOP,DTR1.1,L,BE,C,-164.82155
 ,NIC,BOP,DTR1.1,L,BE,N,-329.31412
 ,NIC,CF,DTR1.1,L,BE,N,165
 ,NIC,IA,DTR1.1,L,BE,N,-0.50743
 ,CDR,EOP,DTR1.1,N,BE,C,0.01059
 ,CDR,BOP,DTR1.1,N,BE,N,0.04427
 ,CDR,CF,DTR1.1,N,BE,N,-0.03368
+,CDRI,EOP,DTR1.1,N,BE,C,0.01059
+,CDRI,BOP,DTR1.1,N,BE,N,0.04427
+,CDRI,CF,DTR1.1,N,BE,N,-0.03368
 ,NIC,EOP,DTR1.1,N,BE,C,-165
 ,NIC,BOP,DTR1.1,N,BE,N,-330
 ,NIC,CF,DTR1.1,N,BE,N,165
 2020,CDR,EOP,DTR1.2,C,BE,C,0.01057
 2020,CDR,BOP,DTR1.2,C,BE,N,0.04415
 2020,CDR,CF,DTR1.2,C,BE,N,-0.03368
+2020,CDRI,EOP,DTR1.2,C,BE,C,0.01057
+2020,CDRI,BOP,DTR1.2,C,BE,N,0.04415
+2020,CDRI,CF,DTR1.2,C,BE,N,-0.03368
 2020,NIC,EOP,DTR1.2,C,BE,C,-164.82155
 2020,NIC,BOP,DTR1.2,C,BE,N,-329.31412
 2020,NIC,CF,DTR1.2,C,BE,N,165
 2020,NIC,IA,DTR1.2,C,BE,N,-0.50743
 2020,CDR,EOP,DTR1.2,L,BE,C,0.01057
 2020,CDR,BOP,DTR1.2,L,BE,N,0.04415
 2020,CDR,CF,DTR1.2,L,BE,N,-0.03368
+2020,CDRI,EOP,DTR1.2,L,BE,C,0.01057
+2020,CDRI,BOP,DTR1.2,L,BE,N,0.04415
+2020,CDRI,CF,DTR1.2,L,BE,N,-0.03368
 2020,NIC,EOP,DTR1.2,L,BE,C,-164.82155
 2020,NIC,BOP,DTR1.2,L,BE,N,-329.31412
 2020,NIC,CF,DTR1.2,L,BE,N,165
 2020,NIC,IA,DTR1.2,L,BE,N,-0.50743
 2020,CDR,EOP,DTR1.2,N,BE,C,0.01059
 2020,CDR,BOP,DTR1.2,N,BE,N,0.04427
 2020,CDR,CF,DTR1.2,N,BE,N,-0.03368
+2020,CDRI,EOP,DTR1.2,N,BE,C,0.01059
```

Comment:
For LIC, there's no CSM to be computed then there's no need to compute and store CDRI